test/spec/binary-leb128.wast

```diff
@@ -1,963 +0,0 @@
-;; Unsigned LEB128 can have non-minimal length
```

Comment:
Is this test invalid? I'd expect a fix to allow more tests to run, so it's not obvious to me why a test was removed.

Reply:
It's removed because we can now run the upstream version at test/spec/testsuite/binary-leb128.wast.